static_frame/core/index_hierarchy.py

```diff
@@ -1031,13 +1032,11 @@ def relabel(self, mapper: RelabelInput) -> 'IndexHierarchy':
 
         if not callable(mapper):
             # if a mapper, it must support both __getitem__ and __contains__
-            getitem = getattr(mapper, 'get')
-
             def gen() -> tp.Iterator[tp.Tuple[tp.Hashable, ...]]:
                 for array in self._blocks.axis_values(axis=1):
                     # as np.ndarray are not hashable, must tuplize
                     label = tuple(array)
-                    yield getitem(label, label)
+                    yield mapper.get(label, label) # type: ignore
```

Comment:
It was to avoid having a `getattr` call, and not being convinced when I was reading it that the performance advantage would be noticeable. I can put it back how it was though